docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
@joaniefromtheblock I think this edit maybe have broken the title and nav bar - it now reads "bonsai-limit-trie-logs" instead of "Reduce storage for Bonsai Tries"